nsiqcppstyle_checker.py

```diff
@@ -983,7 +988,15 @@ def Peek(self):
 
 def ProcessFile(ruleManager, file, data=None):
     #    print file
-    lexer = CppLexerNavigator(file, data)
+    try:
+        lexer = CppLexerNavigator(file, data)
+    except:
```

Comment:
Let's capture `UnicodeDecodeError` only. If there's an OSError, we don't want to skip in that case

Reply:
Done.